Documentation/network/servicemesh/mutual-authentication/mutual-authentication.rst

```diff
@@ -20,13 +20,13 @@ Mutual Authentication (Beta)
 Mutual Authentication and mTLS Background
 #########################################
 
-Mutual Transport Layer Security (mTLS) is a mechanism that ensures the authenticity, integrity, 
+Mutual Transport Layer Security (mTLS) is a mechanism that ensures the authenticity, integrity,
```

Comment:
This file still contains the space changes.

Reply:
I wonder how this file escaped my settings, I made sure to double check

Reply:
Will fix